learn-ocaml.el

```diff
@@ -57,3 +57,28 @@
 		 :fetch t)
        :stderr learn-ocaml-log-buffer
        :buffer learn-ocaml-log-buffer ))
+
+(defun learn-ocaml-file-writter-filter (proc string)
+  (write-region string nil learn-ocaml-temp t)
+  )
+
+(cl-defun learn-ocaml-grade-file (&key id token server dont-submit file)
+  "enables the user to upload a file to the server"
+  (interactive)
+  (set-buffer learn-ocaml-log-buffer)
+  (delete-region (point-min) (point-max))
+  (write-region "" nil learn-ocaml-temp )
+  (make-process
+       :name (concat "upload-" id)
+       :command (learn-ocaml-command-constructor
+		 :token token
+		 :server server
+		 :id id
+		 :dont-submit dont-submit
+		 :file file
+		 )
+       :stderr learn-ocaml-log-buffer
+       :filter #'learn-ocaml-file-writter-filter
+       :sentinel (lambda (proc string) "" (interactive)
```

Comment:
```suggestion
       :sentinel #'(lambda (proc string) "" (interactive)
```

The `lambda` should probably be quoted like this

Reply:
Now-a-days—or rather, now-a-decades—the lambda form sharp-quotes itself, meaning a plain (lambda (x) (* x x)) is identical to the #' version. In fact, you should never quote your lambdas with either quotes. 
http://endlessparentheses.com/get-in-the-habit-of-using-sharp-quote.html